packages/react-fiber-react-18/package.json

```diff
@@ -46,10 +46,12 @@
     "@pixi/react-invariant": "*"
   },
   "devDependencies": {
+    "react-reconciler": "~0.29.0"
+  },
+  "peerDependencies": {
     "react-reconciler": "^0.29.0"
   },
```

Comment:
So the `react-reconciler` is actually currently bundled into the library code (and always has been AFAIK) - so I'm not sure we need it as a peer dep do we?

Reply:
Oh is the rollup config bundling devDependencies and not setting them as external?

Reply:
I'm actually managing `external` libraries manually at the moment, perhaps I should move to your script but still override it in this case

Reply:
`react-three-fiber` actually uses [preconstruct](https://preconstruct.tools/) which looks interesting. Anyway, they don't bundle it in, they just list it as a normal `dependency` so we can do the same